abci/proposals.go

```diff
@@ -12,7 +11,16 @@ import (
 	"github.com/cometbft/cometbft/libs/log"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkmempool "github.com/cosmos/cosmos-sdk/types/mempool"
-	"github.com/skip-mev/pob/mempool"
+	mempool "github.com/skip-mev/pob/mempool"
+)
+
+const (
+	// MinProposalSize is the minimum size of a proposal. Each proposal must contain
+	// at least the auction info.
+	MinProposalSize = 1
```

Comment:
I found myself tripping up over this variable name and its use a few times. I would maybe consider renaming this to something a bit more intuitive or descriptive.

It seems this variable is primarily used to keep a cursor or know the number of "fake" txs injected into a proposal. Perhaps `NumInjectedTxs`? or something like...